### ISSUE TITLE
I was typing text into the ch...

### ISSUE THREAD

**Issue by door43app** [unfoldingWord-dev/ts-desktop#766]
Notes
======
I was typing text into the check screen and the display shrank and expanded back. The chunk I was working on is now blank. I Tim 4:1-2

Environment
======
Environment Key | Value
:--: | :--:
Version |9.1.0
Operating System | Windows_NT
Platform | win32
Release | 10.0.10586
Architecture | x64

Log History
======
```javascript
2016-07-20 18:25:16 I/migrator.js:31:2: Migrated projects
	[]
2016-07-20 18:50:19 I/migrator.js:31:2: Migrated projects
	[
	  "pis_1ti_text_reg"
	]
2016-07-20 19:51:10 I/migrator.js:31:2: Migrated projects
	[
	  "pis_1ti_text_reg"
	]
2016-07-20 22:25:50 I/migrator.js:31:2: Migrated projects
	[
	  "pis_1ti_text_reg"
	]
2016-07-21 22:37:48 I/migrator.js:31:2: Migrated projects
	[
	  "pis_1ti_text_reg"
	]
2016-07-21 22:57:31 I/migrator.js:31:2: Migrated projects
	[
	  "pis_1ti_text_reg"
	]

```

**Comment by cdwhitfield73** [comment]
This issue should be taken care of by the "data loss" fix we did for version 9.2.

**Comment by cckozie** [comment]
Can't really test this.
